test/actions.test.js

```diff
@@ -447,17 +447,33 @@ test('[actions.diff] success', function(assert) {
 
   AWS.stub('CloudFormation', 'describeChangeSet', function(params, callback) {
     polled++;
-
     assert.deepEqual(params, {
       ChangeSetName: changesetId,
-      StackName: 'my-stack'
+      StackName: 'my-stack',
+      NextToken: undefined
     }, 'describeChangeSet expected parameters');
 
-    if (polled === 1) return callback(null, { Status: 'CREATE_IN_PROGRESS' });
+    if (polled === 1) return callback(null, { ChangeSetName: changesetId,
+      ChangeSetId: 'changeset:arn1',
+      StackId: 'arn:aws:cloudformation:us-east-1:123456789012:stack/my-stack/be3aa370-5b64-11e6-a232-500c217dbe62',
+      StackName: 'my-stack',
+      ExecutionStatus: 'AVAILABLE',
+      Status: 'CREATE_IN_PROGRESS',
+      Changes: [{
+        Type: 'Resource',
+        ResourceChange: {
+          Action: 'Modify',
+          LogicalResourceId: 'Topic',
+          PhysicalResourceId: 'arn:aws:sns:us-east-1:123456789012:another-stack-Topic-ABCDEFGHIJKL',
+          ResourceType: 'AWS::SNS::Topic',
+          Replacement: 'False'
+        }
+      }]
+    });
```

Comment:
i'd love to see two more response types:

- [ ] having no .`Changes` returned
- [ ] having a `NextToken` returned